client/src/components/questionsAnswers/QuestionForm.jsx

```diff
@@ -114,41 +114,63 @@ class QuestionForm extends React.Component {
 
   showForm() {
     return (
-      <div className="form-box">
+      <div className="qboxcenter">
         <form>
-        <h1>Ask Your Question</h1>
-          <label>
-            Enter UserName*
-            <input value={this.state.name}
+          <button type="submit" className='X' onClick={this.onCloseForm}>X</button>
+          <h1>Ask Your Question</h1>
+
+          <label className="field field_v1">
+            <input
+              value={this.state.name}
+              className="field__input"
               onChange={this.onChangeName}
               type="text"
               name="name"
-              placeholder="Example: jackson11!**"
+              placeholder="Example: Isaac123!**"
             />
+            <span class="field__label-wrap">
+              <span class="field__label">Enter UserName*</span>
+            </span>
           </label>
 
-          <label>
-            Enter Email*
-            <input value={this.state.email}
+          < br/>
+
+          <label className="field field_v1">
+            <input
+              value={this.state.email}
+              className="field__input"
               onChange={this.onChangeEmail}
               name="email"
               type="text"
               placeholder="Why did you like the product or not**"
               pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/"
```

Comment:
I like how you made a different line for each item for your label tag. I think I'm going to do this too, makes it easier to read